frontend/src/components/regions/UserActions/ActivityLog.jsx

```diff
@@ -3,10 +3,15 @@ import Typography from "@material-ui/core/Typography";
 import Paper from "@material-ui/core/Paper";
 import TodayIcon from "@material-ui/icons/Today";
 import Chip from "@material-ui/core/Chip";
-import Grid from "@material-ui/core/Grid";
 import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles((theme) => ({
+  paper: {
+    width: "50rem",
+    ["@media (max-width:620px)"]: {
+      fontSize: "1rem",
+    },
```

Comment:
One tiny thing, this line generates a warning. Here's how I fixed it:
![image](https://user-images.githubusercontent.com/54069197/192995675-0746077e-3921-4b1f-ab6e-2b8c0bdf56b1.png)